index.html

```diff
@@ -34,7 +34,7 @@
 								<li><a href="#">Join</a></li>
 								<li><a href="#">Sponsor</a></li>
 								<li><a href="#">News</a></li>
-								<li><a href="#">CC Campaign</a></li>
+								<li><a href="#" class="campain">CC Campaign</a></li>
```

Comment:
> Please as the last reviewer said ensure that each of your `speaker cards` is well aligned as in the original design.
> 
> Your Implementation	Design
> ![image](https://user-images.githubusercontent.com/10332499/163295130-c5fe7787-0ef8-43bc-b5a7-a752a318897a.png)	![image](https://user-images.githubusercontent.com/10332499/163295041-c868a542-19b5-48b0-9790-08193cc8ee1c.png)

Reply:
Hello @MeqdamAlqudah, kindly note that I updated the speaker cards and the  screenshot you provided is not my current design